contracts/FeeSettings.sol

```diff
@@ -49,16 +49,19 @@ contract FeeSettings is
     uint32 public privateOfferFeeDenominator;
 
     /// address the token fees have to be paid to
-    address public tokenFeeCollector;
+    address private tokenFeeCollectorAddress;
```

Comment:
why is this renaming necessary? I think the old version is cleaner

Reply:
It is necessary because now I manually write the getter function `tokenFeeCollector()` (the name of which is defined in the interface). As this function is a variable, there can be no other variable with the same name.
https://ethereum.stackexchange.com/questions/142145/how-do-i-override-the-auto-generated-getter-of-a-public-variable
https://docs.soliditylang.org/en/v0.8.17/types.html?#function-types

This corresponds with my experience from tinkering around.

Reply:
why are they private after the name change?